### DISCUSSION TITLE
Google analytics DATA missing on Plausible API

### DISCUSSION THREAD

**Discussion by NoeGRANGE** [plausible/analytics#2154] [Anything Else]
Hello,

I'm trying to make a custom study of Plausible stats but I can't find my Google analytics data while requesting the API.

<img width="178" alt="image" src="https://user-images.githubusercontent.com/91610312/187632122-6a08d0a5-3920-474b-a8e1-5e7310a27301.png">

It returns me 0 for every query before the date I switched to Plausible.
Am I requesting badly or does the data isn't currently available by the API?

Thank you in advance !

**Comment by metmarkosaric** [comment]
hi @NoeGRANGE! it isn't possible to export the data imported from Google Analytics using our usual methods, such as the [CSV export](https://plausible.io/docs/export-stats) and the [stats API](https://plausible.io/docs/stats-api). this and the other differences between native and imported data are listed here https://plausible.io/docs/google-analytics-import

**Reply by NoeGRANGE** [reply]
Ok, I hadn't read that thanks again 👍​

**Comment by metmarkosaric** [comment]
**Update:** You can now export your imported data using our regular export methods (CSV and stats API). See how:

CSV export: https://plausible.io/docs/export-stats#export-specific-time-ranges-or-audience-segments
Stats API: https://plausible.io/docs/stats-api#imported-stats
